src/components/card/AttachmentList.vue

```diff
@@ -79,6 +79,9 @@
 					<ActionLink v-if="attachment.extendedData.fileid" icon="icon-folder" :href="internalLink(attachment)">
 						{{ t('deck', 'Show in Files') }}
 					</ActionLink>
+					<ActionLink v-if="attachment.extendedData.fileid" icon="icon-download" :href="downloadLink(attachment)">
```

Comment:
How about adding `:download="attachment.name"` in order to really make it a download link?

Reply:
Actually, just adding `download` is enough. I tested it locally and it works as expected.

Reply:
Shall I push the change or do you take care of it @juliushaertl?

Reply:
Added and pushed :)